e2e/cypress/support/index.ts

```diff
@@ -14,7 +14,16 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
+import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+before(function () {
+  if (Cypress.env('ADMIN_TOKEN') === undefined || Cypress.env('ADMIN_TOKEN') === '') {
+    cy.adminLogin('/').then(() => {
+      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      cy.generateAdminToken(admin.id_token);
+    });
+  }
+});
```

Comment:
Oh interesting! Can you throw a comment about how this works in our cypress readme? I could see this feeling a bit magical if you weren't aware it existed. I like it though.